docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -74,25 +78,27 @@ Finally start your plugin in development mode. Go to your plugin root directory
    yarn start
    ```
 
-   2.2 Or with docker-compose, in your plugin directory, run:
+   b. Or, with docker-compose in your plugin directory, run:
```

Comment:
the second comma is important, docker-compose will be in your plugin directory, so a user may go yup thats my situation, but they need to issue the command from within the plugin directory, not the grafana source code directory as above

Reply:
I can't see any update or change related to this?

Reply:
got it